exercises/bracket-push/examples/success-standard/src/BracketPush.hs

```diff
@@ -0,0 +1,36 @@
+module BracketPush (isPaired) where
```

Comment:
I would just call the module `Brackets`. the `Push` is too much of an implementation detail and I don't even think it should go in the name: https://github.com/exercism/x-common/issues/693

I would maybe call the function `arePaired` so that can read like `Brackets.arePaired`, and this makes sense since it handles multiple brackets not just one pair.